backend/core/diagnose.py

```
@@ -367,10 +367,29 @@ def run_diagnostics(include_network: bool = True, deep: bool = False) -> dict:
+            active = module.active_backend_id()
+            row = next((item for item in module.list_backends() if item.get("id") == active), None)
+            if row is not None:
+                engine_execution.append({
+                    "family": family,
+                    "engine_id": active,
+                    **row["execution_evidence"],
+                })
+    except Exception:
+        # Evidence supplements the health checks; a registry failure is already
+        # represented by the engine check and must not break the bundle.
+        pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)